internal/experiment/experiment.go

```diff
@@ -32,7 +32,11 @@ type Experiment string
 const (
 	// BoltFrameworks experiment adds CLI support for Bolt JavaScript & Bolt Python.
 	// These frameworks will be introducing remote function support.
-	BoltFrameworks = "bolt"
+	BoltFrameworks Experiment = "bolt"
```

Comment:
note: Updated to be cast as an `Experiment` instead of `string`

Reply:
Woah great catch... At first I thought this was an extra space 😴